ozaria/site/components/interactive/PageInteractiveEditor/index.vue

```diff
@@ -0,0 +1,208 @@
+<script>
```

Comment:
This page looks good.

Reading over this I know you used some of the Cinematic Editor as inspiration. I wonder how you felt trying to jam the two pages into this single page. (The list view, and the editor view).
I feel there is probably a list vue that can be pulled out and used instead of having the page handle both list and individual interactive/cinematic.

If it works that is all that matters! :) Great stuff!

Reply:
I completely agree with you! I had the same feeling while working on this that we should have a separate page to handle to list and an individual list item.
We should definitely update this design in the future!